### DISCUSSION TITLE
Normalization of Velocity Profile for Use in Another Model IC

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#23347] [Q&A Modules: Navier-Stokes] [answered]
In order to have an initial guess near the expected final solution, I am solving a problem domain for the velocity profile using fluid properties that provide a more easily converged solution. I would like to then take that solution and normalize it in relation to the maximum velocity. Is there a way for me to then use, the stored solution, normalize it, then provide a multiplier for the expected max velocity based on the actual fluid properties to be used, to then be the initial conditions based on the normalized and multiplied velocity profile?

**Comment by GiudGiud** [comment, marked as answer]
Hello

yes you can do that. 
you ll want to save the variables in an exodus output for example. Then load them into auxiliary variables. Then use an auxkernel (likely ParsedAux) to perform the operations you wanton that field and save the output in another auxvariable.

you can then use this other auxiliary variable for the initialization by loading it from a file again. 

Basically you ll need an intermediate input file to perform these operations 

If you’re comfortable with MultiApps you could do this in a multi app executed on INITIAL 

Guillaume

**Reply by TLWise** [reply]
If my initial solution is solved on a course mesh, Is it possible to interpolate nearest node/element solutions onto a finer mesh using the normalized values determined from the course mesh solution as the initial guess for the fine mesh problem solve?

**Reply by GiudGiud** [reply]
Sure. 
You can look at using a MultiApp for that. Load your solution in the multiapp, then the transfers will do this projection from one mesh to the other for you
